### DISCUSSION TITLE
Export to Game Engine

### DISCUSSION THREAD

**Discussion by bqkhiem96** [domlysz/BlenderGIS#490] [General]
Is there any way to export this to a game engine and treat it as a landscape component?

**Comment by domlysz** [comment]
http://wedesignvirtual.com/which-file-formats-does-blender-support/

**Comment by rhclayto** [comment]
You could export it as a mesh that your game engine could import. I only know Unity. In Unity you can use an asset called Mesh to Terrain to convert the imported mesh to a terrain.
